src/Role/ReadModel/Permissions/Doctrine/UserPermissionsReadRepository.php

```diff
@@ -31,7 +31,7 @@ public function __construct(
     /**
      * @return Permission[]
      */
-    public function getPermissions(StringLiteral $userId): array
+    public function getPermissions(string $userId): array
```

Comment:
I think we can easily refactor the constructor of `UserPermissionsReadRepository` to use a native `string` instead of the obsolete `StringLiteral` class. The `StringLiteral` is certainly a class you can refactor away in the scope of boy scouting.